src/gui/application.cpp

```diff
@@ -770,24 +774,38 @@ void Application::setupTranslations()
             const QString qtTrPath = QLibraryInfo::location(QLibraryInfo::TranslationsPath);
             const QString qtTrFile = QLatin1String("qt_") + lang;
             const QString qtBaseTrFile = QLatin1String("qtbase_") + lang;
+            qCDebug(lcApplication()) << "load" << qtTrFile << "from" << qtTrPath;
             if (!qtTranslator->load(qtTrFile, qtTrPath)) {
+                qCDebug(lcApplication()) << "load" << qtTrFile << "from" << trPath;
                 if (!qtTranslator->load(qtTrFile, trPath)) {
+                    qCDebug(lcApplication()) << "load" << qtBaseTrFile << "from" << qtTrPath;
```

Comment:
do we really want to keep all these debug messages?

Reply:
removed now